ansible/group_vars/omero-hosts.yml

```diff
@@ -83,6 +83,20 @@ omero_user_system: omero-server
 ######################################################################
 # openmicroscopy.omero-web role vars
 
+idr_omero_web_public_url_filters_webclient_exclude:
```

Comment:
Is this something for the role itself?

Reply:
I don't know. Do we need a filter with `https://docs.openmicroscopy.org/omero/5.4.0/sysadmins/config.html#std:property-omero.web.public.get_only`?